core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -122,7 +126,7 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
         } else {
           logInfo("Registered executor: " + executorRef + " with ID " + executorId)
           context.reply(RegisteredExecutor)
-
+          latestTokens.foreach(x => context.reply(NewTokens(x)))
```

Comment:
I actually tried doing that, but it failed a huge number of tests and thought it was perhaps not worth that risk. 

With the new RPC thing, I was not sure if calling send from inside this method would break something somewhere. Really, yes, it is `send` we want to call here.